csr2transmart/mappers/observation_mapper.py

```diff
@@ -224,7 +224,8 @@ def map_study_registry_observations(self):
                                        'Failed to create observation for individual study with id: {}.'
                                        .format(ind_study.study_id, ind_study.individual_id))
             entity_type_to_id = {'Individual': ind_study.individual_id}
```

Comment:
I wonder if we could also reuse `self.get_ref_entity_name_to_ref_field_value(entity, entity_type)` here

Reply:
But I don't know if we want to add a Study modifier.... that would probably conflict with the built-in study dimension.